src/vue-components/panel.vue

```diff
@@ -684,19 +684,6 @@
         stroke: $alpheios-link-hover-color;
     }
 
-    .alpheios-panel__body {
```

Comment:
the panel still resize itself correctly with the flex styles gone?

Reply:
There is no such class in html layout.
I have checked the code and didn't find it. 
It seems to me that on some iteration there were chnages to layout and this class is outdated.